docs/en/manuals/project-settings.md

```diff
@@ -321,6 +321,12 @@ If specified, use this info.plist file when bundling your app.
 #### Entitlements
 If specified, can override wildcard entitlements defined in the supplied provisioning profile (.entitlements, .xcent, .plist).
 
+#### Default Language
+This language uses if application doesn't have user's preferred language in `Localizations` list (see [CFBundleDevelopmentRegion](https://developer.apple.com/library/archive/documentation/General/Reference/InfoPlistKeyReference/Articles/CoreFoundationKeys.html#//apple_ref/doc/uid/20001431-130430)). Use the two-letter ISO 639-1 standard if preferred language available there or the three-letter ISO 639-2.
```

Comment:
"This language uses if application" -> "The language used if the application"

"if preferred language available" -> "If preferred language is available"

Reply:
Thx, Done!